static/js/app.js

```diff
@@ -573,3 +606,79 @@ $(document).ready(function() {
         $('#beta-banner').get(0).expand();
     }
 });
+
+/* ==========================================================================
+   Alert
+   ========================================================================== */
+
+(function( $ ) {
+
+    // Alert constructor.
+    // element {Element} Dom Element.
+    // @options {Object} Object used to customize Alert.
+    // Returns Alert instance.
+    var Alert = function(element, options) {
+        this.$element  = $(element);
+        this.options   = $.extend(true, {}, this.defaults, options);
+        this.init();
+    };
+
+    // Plugin methods and shared properties
+    Alert.prototype = {
+
+        constructor: Alert,
+
+        defaults: {
+            validateSelector : '.js-validate_form-not-empty',
+            easing           : 'linear' 
+        },
+
+        init: function() {
+            var parentForm = this.$element.parents(this.defaults.validateSelector);
+
+            if(parentForm.length === 1){
+                 parentForm.on('form:validate:empty', $.proxy(this.show, this));
+                 parentForm.on('form:validate:not_empty', $.proxy(this.hide, this));
+            }
+            return this;
+        },
+
+        destroy: function() {
+            this.$element.removeData();
+            return this;
+        },
+
+        show: function() {
+            this.$element.slideDown({
+                 easing: this.defaults.easing
+            });
+            return this;
+        },
+
+        hide: function() {
+            this.$element.slideUp({
+                 easing: this.defaults.easing
+            });
+            return this;
+        }
+    }
+
+    // Create the jQuery Alert plugin.
+    // @options {Object} Object used to customize Alert.
+    // Returns jQuery element.
+    $.fn.Alert = function(options) {
+        return this.each(function() {
+          var $this = $(this);
+          $this.data('plugin', new Alert(this, options));
+        });
+    };
+
+}( jQuery ));
+
+/* ==========================================================================
+   Alert Initialization
+   ========================================================================== */
+
+$(document).ready(function() {
+   $('.alert').Alert();
+});
```

Comment:
Run `grunt lintjs` and fix the errors that you can. Don't worry about the warnings unless you want to fix those. For example instead of `$('.alert').Alert();` perhaps have it as `$('.alert').createAlert();`, since the `Alert` attached to jQuery isn't a constructor.